### DISCUSSION TITLE
Thinking Cap #4 💡

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#45] [Thinking Caps]
### Reminder: Responses submitted after Sunday 3pm will not be eligible for awards.

> ## What do you think about design justice? Would open-source projects benefit from considering it? How might you apply Design Justice in your designs going forward?

The reading on [Design Justice](https://albertkun.github.io/24SU-ASIAAM-191A/materials/readings/design_justice.pdf) must be completed before responding to this thinking cap.

## Submission
Make a new comment below and respond to two students comment before class begins.

**Comment by stinest** [comment]
I believe design justice is an important concept that is necessary in promoting inclusivity in any and all creative processes. As important as technology is, design as its framework deserves just as much attention in ethical considerations, which is why I think open-source projects would benefit significantly from it. Since open-source projects often source from diverse communities, considering design justice can guarantee the tools its developing are accessible and beneficial to as many users as possible. By "[centering] the voices of those who are directly impacted by [its] outcomes" (pg. 2), it is positively prioritizing the needs and perspectives of potentially marginalized groups. 

Going forward and especially in our final project, I hope to apply these same concepts in our survey design and in overall web development. Especially since the community plays a large part in our data sourcing, I want to ensure that those directly impacted by design outcomes have a significant say in the process by community consultations and asking for feedback.

**Reply by jasminereyz** [reply]
I completely agree with your perspective, I also feel that design justice is necessary and can be useful when considering ethics. I also like how we both had similar quotes that we resonated with as I also feel that we should prioritize the needs of marginalized groups.

**Reply by Keiya-Wada** [reply]
I completely agree with what you said, especially with the statement that the design of technologies deserves more attention than it currently does. The convenience of technology won't necessarily exist without the ethical considerations put behind them, and without a good design thought behind the technology, it is likely that the tech will face a lot of problems later on.

**Reply by nesslee123** [reply]
Really well written response! I completely agree with your point about the importance of Design Justice in emphasizing the necessity for inclusivity because often marginalized communities are not considered in design processes which further oppresses these groups of individuals. Also, I enjoyed how you mentioned that you would be implementing Design Justice in your final project because it shows your efforts to create an inclusive environment among your survey participants and throughout your project, in general.

**Comment by jasminereyz** [comment]
I think design justice is a crucial aspect to web development and ensuring that we are accessible and inclusive. The reading mentioned, knowing who "you are designing for", knowing what values you have while designing and I think that stuck to me the most. Especially in terms of our final project, I think it is easy to get lost in creating the design that we may forget who we are even designing for, many of us are designing for organizations and not just a good grade in this class. This leads me into open-source designs and how tremendously they can benefit from this method. I think because open source projects are meant to be fully accessible to anyone, ensuring that intersectionality and ethics are kept true, can reduce the several problems we see in not only web design but map making.

**Reply by Keiya-Wada** [reply]
I completely agree with what you said! Although my intentions were not just to get a good grade, I have already experienced creating survey questions that may have lacked some care and respect towards the community taking the survey, so thinking about design justice will allow us to minimize those kind of problems just like you said.

**Reply by kianaptran** [reply]
Yes! It is so easy to get caught up with designing projects to serve a specific idea you initially envisioned and disregard the whole reason you wanted to design a project in the first place (serving the community)! The philosophy of constantly referring back to whether it's designed FOR the people is a great way to ensure design justice.

**Reply by pwinnie2026** [reply]
I agree! Your point on how it can be easy to get lost in the design process that we can lose sight of who we are designing it for is something that I have had to pay attention to during this class. Especially with my project, since I am coming from an outsider perspective, I recognize that it can be hard initially without consulting someone within the space I am trying to design with. Open-source designs can benefit a lot from these design justice principles. I agree that whether it is web design or map making or etc., opening these projects to the community and allowing them to take ownership of their work is a great way for it to stay true to who you are designing it for.

**Comment by Keiya-Wada** [comment]
I think design justice is a very important field, as it introduces a sense of inclusivity by making people think about how they can achieve such a status. Especially because society society is moving towards valuing the concept of diversity, design justice has become prominent in recent years. Open-source projects would definitely benefit from design justice, as open-source projects are easily accessible to anyone, and we want to really think about how to design the map and questions in order for the map to not become a tool to limit certain perspectives. Because I have already experienced accidentally phrasing questions in a way that forces assumptions onto the surveys, I want to keep this concept of design justice in mind and make create a project that truly empowers the community's voice.

**Reply by kianaptran** [reply]
I agree entirely, Keiya! I also had the issue of phrasing questions framed to garner specific responses instead of allowing the community to speak on the problems they face themselves! I think with a more design justice approach, it will be better suited to serve the community than have the community serve our project.

**Reply by NellieNguyen149** [reply]
I really enjoy how you bring up inclusivity in your discussion about design justice! There is a big connection that can be made between open-source projects and design justice because of similar goals and mindsets, and yeah, I've also had my fair share of wording questions in a way that could be improved on to be more inclusive/mindful of all community groups. Thank you for sharing!

**Reply by jasminereyz** [reply]
I also agree with your perspective! My partner and I also ran into the same problem because we did not want to mislead anyone with our questions. I think it is great that this article has shown us to keep design justice in mind to ensure our projects are genuinely empowering.

**Reply by kimada62** [reply]
Hey Keiya,
I know we both worked hard to not make assumptions with our surveys throughout our project, which I think was successful and unsuccessful at times through our trial and error. It is a difficult balance to strike, because obviously our primary goal was always to lift the voices of Japanese Americans within the community. I've been studying entrepreneurship more since our class and it felt reminiscent of the way that startup owners have to keep in mind to observe and find the user needs in the early stage of exploration. In both entrepreneurship as well as design justice, we are not crafting an idea of what people need in our head. We have to observe and ask people the right questions that will organically reveal the problem. The difference is that with entrepreneurship, we make a business to make a difference. In design justice, the research needs to be presented to lift whatever concerns arise in order to urge society to make a difference.

**Comment by kianaptran** [comment]
Design justice is a very important progression towards inclusivity within science and technology. Personally, I find the concept of design justice very inspiring if it can be brought into practice entirely. What I mean by this is that the framework, as described in the ideal process, is highly idealized. As long as there are corporations that are fueled by capitalism, a completely non-exploitative and community-based process is not yet achievable. That being said, as a "living document" that is being worked towards, once we can achieve this framework, it would be highly beneficial to serve the community better while having the community help improve science and technology to be more inclusive and progressive. As open-source is accessible to anyone, these projects would benefit from considering design justice, encouraging more individuals to be involved in the design process, and increasing technological literacy. One point that was mentioned in the article that was a part of Design Justice's objectives is inclusivity "towards equity in the tech workforce [by having] organizations that build the design skills of more women, POC, and LGBTQ folks" (Constanza-Chock 8). Programs that help educate marginalized and underrepresented groups to become technologically literate would improve open-source inclusivity (win-win). Moving forward, I hope with my group project, we will be able to integrate the idea of community-based design (having a listen rather than solve approach to our topic), which will benefit the community and the usefulness of our application.

**Reply by sjeyakum** [reply]
I agree that while design justice may seem idealistic as a framework, it becomes more practical and relevant when viewed as a dynamic movement or "living document," as you emphasize in your post. I think this idea can also shift the perspective on design justice from a checkbox to a collaborative goal that designers and communities can work towards. Your concluding sentences also highlight this concept as you stress the importance of ensuring the community understands the different elements and technologies used in a design so the project becomes truly beneficial. I also hope that my group project can address this aspect of design justice!

**Reply by sachigill** [reply]
I thought your point regarding the concept of design justice being highly idealized, to be very interesting and something I resonate with myself.  Even with other types of frameworks that strive for equity and inclusivity, it can be difficult to truly implement these given the current constraints of being in a capitalistic society. It feels like there is always a limit of what can be achieved due to the never ending conflict between capitalism and these frameworks, which can be kind of discouraging especially when you are hoping for social progress and change.

**Comment by kchhoeu** [comment]
I think design justice is really important for contributing to accessibility and intersectionality. Inclusivity is important for all field, but especially in the science and technology sector because it deals with data of so many people. The reading mentioned "In Black Feminist Thought, Collins notes that “People experience and resist oppression on three levels: the level of personal biography; the group or community level of the cultural context created by race, class, and gender; and the systemic level of social institutions." I read this the Black Feminist Thought for two of my other classes last year, and I think it does a great job at describing why inclusion and accessibility is important, and to relate it to the class web development would benefit from design justice a lot, especially in the sector of community-based designs

**Reply by NellieNguyen149** [reply]
Intersectionality is an interesting concept that relates to both open-sourced projects and design justice a lot, so that's really cool that you mention it! One aspect of intersectionality that I identify with the most is how it acknowledges that fields of studies and social issues are usually multifaceted and not one dimensional. Also, that's super cool to hear that you've read the Black Feminist Thought in your other classes, which to be honest is pretty representative of how social change as a whole is intersectional becuase of how applicable it is to all types of courses. Thank you for sharing!

**Reply by nesslee123** [reply]
I really enjoyed reading your response! I completely agree with your point regarding the importance of Design Justice in regards to accessibility and intersectionality because Design Justice is necessary for addressing the issue of design practices that further oppress marginalized communities. I also agree that web development would benefit from Design Justice because it would allow for it to be more encompassing and inclusive of individuals who face oppression and inequality.

**Comment by NellieNguyen149** [comment]
Based on the introduction to the paper where it defines the scope of the Design Justice Principles, specifically with the gathering of community designers and organizers to challenge institutions and groups in power. I believe these aspects align well with open-source projects where they have similar goals of accessibility to the general public and ensuring transparency. Open-source projects are what allow people to know and understand their intentions and the changes they make without anything being hidden behind the walls of close-sourced projects, and from this reading, design justice seeks to empower the community in a similar way with a lens of creativity. 

One portion of the reading that stood out to me in relation to my own designs/project in this class was the "Users" section. "Who do design for/with? ...designers most frequently assume that the unmarked user has access to a number of very powerful privileges... There is growing awareness of this problem, and a number of designers, projects, events, and communities of practice who are attempting to address it through intentional focus on designing for, or with, communities who are usually invisibilized in the world of technology" (pg. 9). For me personally, one of my biggest takeaways from this class is focusing on the community when deciding an assignment or project objective and focusing. It also includes understanding that current data and knowledge are not always representative of everyone due to limitations and barriers to voicing their opinions. Design justice utilizes the same mindset as open-sourced projects while serving as an art form, which I believe is a useful way to provide a unique and powerful  visualization of community-based empowerment projects.

**Reply by sjeyakum** [reply]
I completely agree with you, Nellie! I also appreciate how this course consistently prioritizes community engagement. From my own experience with the group project, I have come to understand that there is significant difference between simply claiming a project is community-focused and genuinely ensuring it serves the needs of a community. I also find your perspective on linking design justice to an art form to visualize community-centered projects interesting as I think there is a lot of potential for design justice to creatively and meaningfully integrate into open-source initiatives!

**Reply by giaari15** [reply]
I appreciate your understanding of projects as art-forms and fully agree with you. I wonder if intentional community also has its downsides, as we should be critical of everything in my opinion. For example, the reading mainly builds off of the framework of intersectionality, but I feel that intersectionality is a bit individual-based and does not take a strict anti-imperial and anti-capitalist stance. Recently I have been wondering about how to view a community and our own perspectives in understanding who and what a ocmmunity is.

**Comment by sjeyakum** [comment]
After reading Costanza-Chock's article, I see design justice as a paradigm shift towards more accessible, inclusive, and _**sustainable**_ goals. I think it's easy to forget that justice can look very different depending on the context and perspective, so defining design justice with static guidelines would overlook important nuances in social issues and marginalized groups. This approach would undermine the very idea of fostering change and working towards justice by focusing on a more rigid problem-solution mindset that only works in a single moment, failing to consider the evolving needs of the community. As such, I appreciated Costanza-Chock's tentative definition that design justice is "a growing social movement" that aims to address equitable distribution, meaningful participation, and community recognition (5). I think the emphasis on growth here is crucial because perspectives and applications regarding design justice should be allowed to change so that we can learn and adapt alongside it. For instance, the reading mentions Kimberlé Crenshaw's intersectionality concept, an idea that was originally dismissed before it sparked increased awareness about power, privilege, and personhood and led to a "paradigm shift that is still unfolding" (4). 

For these reasons of dynamism and growth, I believe open-source projects would greatly benefit from considering design justice because they have long-term viability and the potential to engage in community collaboration that respects the past, empowers the present, and supports future generations. That's why I think the tentative definition of design justice should also include how the growing social movement promotes **_sustainability_** so that designs not only address immediate issues but also contribute to long-term equity, which aligns with a more holistic view of design justice, in my opinion. Open-source projects can work towards this by adopting a design justice mindset and giving ownership to local communities so they have the tools and resources to become resilient and benefit from the design without the unjust pressure of losing it due to shifts in third-party control, priorities, values, etc. I think this aspect of inclusivity and sustainability in design justice is something I want to apply to projects I am involved with, including our final project for this class. I believe it's important for my team and me to understand that when we work on a project for a specific community, they must have ownership of it. It would be wrong for us to retain ownership while assuming the project is genuinely beneficial and sustainable for the community if there is a clear imbalance in power and authority.

**Reply by stinest** [reply]
This is a really well-written response, and I think you described beautifully why design justice- as well as all forms of justice- must be a priority if we want to address the community in an inclusive way. I also really loved your last point in giving ownership to the community if that is who the project is intended for to avoid a power imbalance.

**Reply by sachigill** [reply]
I really enjoyed reading your response, especially regarding your mentioning of design justice not being something static. Depending on the community or social context, there isn't just one generic framework that can be applied to these processes, and it would only be harmful to the communities you may aim to aid if you use static guidelines, without taking all of these different factors into account. Design justice must be allowed to adapt to new concepts and the ever changing goals and needs of communities if you truly want them to benefit.

**Comment by sachigill** [comment]
Design justice is a really important framework that should be applied when going through the design process, as it’s important to consider all the ethical aspects involved. I believe open source projects would benefit, as encouraging community engagement and fostering a participatory approach along with active collaboration, will help produce more equitable designs and address any biases or inequalities that may arise, rather than if only the designers were solely in charge of these decisions. 

Making sure to consider the needs of all populations in your design is important, like for example designing a website that allows for screen reader accessibility, ensuring those with blindness or vision impairments can access these tools, or considering what colors may be best for those with colorblindness. In the design testing, it becomes vital to include all of these populations in that process, so that any potential accessibility or ethical issue can be identified. By not applying design justice principles in these processes, you are actively excluding populations from accessing your designs, further perpetuating the marginalization and oppression of these communities. Even in areas like urban planning, many of these spaces are often designed catering to the wants of those who are wealthier, which can ultimately lead to the displacement of lower-income communities or long-time residents of these areas, which is why it is essential to increase the participatory design efforts being made, to include these communities, especially since they are the ones who will be facing the consequences or impacts of these design decisions. 

In regards to my own designs, I hope to apply these principles when working on our project, making sure to design surveys that don’t insert my own biases, as well as ensuring that the overall platform benefits the community, and allows their voices and thoughts to be uplifted.

**Reply by pwinnie2026** [reply]
I agree with you that it is important to consider the needs of all populations. This was something I admittedly did not address in my own response, but it is definitely a belief I do share. Design testing is very much an iterative process, which allows for you to go back to the drawing board when your work is not accessible or ethical for any particular reason. Especially while you may have a certain community in mind, it is important to not actively exclude populations from accessing your work. There is oftentimes intersectionality and overlapping identities to consider. For this reason (as it is a complex enough task for just one designer or one developer to consider), participatory design and community-based open-sourcing are all means to help designs work towards inclusivity and accessibilty.

**Reply by zhamic7** [reply]
I wholeheartedly agree with your point about screen reader accessibility! I've had some experience with designing websites with other people in the past; I thought making the website readable to everyone should be the number one priority since I can't imagine why else one would make a website in the first place, but I learned that there are actually many people who don't care at all and prioritize other factors like "aesthetics."

**Reply by kimada62** [reply]
Hello with a very late response!
Wow I thought you made a great point with addressing the importance of who can address your platform. I also did not include the factors of color choice and and screen reader accessibility. That is something I will research more and look to address in my future designs.

**Comment by zhamic7** [comment]
Design justice is one way to challenge the dominant discourse and amplify the experiences of those who have been previously ignored in technological and software development. The products we use in our daily lives shape our beliefs about what design decisions are more important (i.e., what user experiences are more important) and dictate what we find to be a "good" design versus a bad one. In return, designers reinforce the same biases and prejudices they learned from other designs if they fail to consider design justice principles in the process of product creation. Because of this, I believe that open-source projects would benefit from design justice.  Compared to commercial products, open-source projects are community-based and not as profit-driven, which means they have more potential to be inclusive and empowering. However, they currently fall short of this expectation due to the lack of emphasis on design justice in this community: the groups (e.g. engineers) that contribute to open-source projects are still mostly cisgender men that are White or Asian, meaning that many of the design assumptions and decisions made are largely shaped by the experiences of said groups. In my own life, I have learned from my digital humanities classes about the importance of making software products accessible to all users, but I believe that moving forward, it is also critical that I first challenge the assumptions I have about the purpose of a product itself and critically analyze whether or not I am just reinforcing mainstream biases and current power structures in the design process.

**Reply by stinest** [reply]
Yes! I completely agree with every point you made. As something used so frequently in our lives, design principles often fall under a loop where it continuously reinforces negative biases, which is why design justice is so important to recognize, but is also why it is so important that we as users need to critically assess in using any software or technology.

**Comment by pwinnie2026** [comment]
I believe that design justice is an important concept, but one that is multi-facetted and nuanced depending on the context. Overall, I recognize that design justice is striving for more collaborative, community-oriented design processes that are sustainable to the community, and overall more accessible and inclusive. More specifically, the verbiage that Costanza-Chock writes about design justice are indicative of this iterative, "living" process. Having "tentative" definitions in a "growing" social movement is important in illustrating this iterative process to center around the community. I also enjoyed reading about the importance of using what already works at the community level before looking towards newer design solutions. Especially with technology developing rapidly, it is important to recognize that you should be designing a product (or anything at all) with users in mind. Real people and real communities who may have build precedents in designing solutions that may not be considered "official" or from an expert. This discussion about designers acting as facilitators rather than experts is important as the difference in mindset can reflect the type of contributions and lived experiences you are centering in your work. 

Open-source projects, I believe, would benefit from considering design justice as a lot of open-sourcing is already reliant on community collaboration. Still, open-source projects exist within a larger context as noted by Costanza-Chock. Designers, programmers, etc. are mostly represented by cisgender White or Asian men, which can translate to the population of those contributing to open-source projects. Especially when projects are being developed and designed from an outsider perspective, design justice should be an important pillar that centers it back onto the community and allows community members to continue the design process. In my own projects and design work outside of this class, one quote by Costanza-Chock stood out to me about "default" imagined users and how designers may make a number of assumptions under this "default" label. I hope to take these principles into any project that I work on, such as for this class. I find that it can be easy to detach yourself from your work when in a corporate or academic sense. Even in hackathons that use problem-solution approaches, it is important to critically assess who (what community) you are building this for. What information do you know about them, who can you talk to within the community, and where are the gaps in what is accessible through research/data and the real lived experiences of your intended users.

**Reply by giaari15** [reply]
I agree with your perspective especially with emphasizing the point that these people are facilitators not experts. I believe even their roles as facilitators be questioned and critiqued as well, like if they are an "outsider" why should this person be designing for a community. I also would love to hear more about getting detached in an academic sense, which I understand and guilty of as well.

**Comment by giaari15** [comment]
The application of design justice is important to continue to counteract the consequences of systemic oppression. I do believe adopting a more straightforward anticapitalist framework is more fruitful to achieving the goal of design justice as it would mirror how intersectionality is applied where the representation matters and not the practice. Recently I have been exposed to a lot of critiques on intersectionality because the recent posts of Crenshaw idealizing Kamala Harris, therefore while I still find it important for consideration, I wonder if the inclusive goals of design justice are also too dependent on individual identity. Furthermore, design justice and intersectionality both aim to better marginalized groups of people position within a capitalist society, not liberate them even though the article abstract stresses collective liberation. Still, I believe it is a slight step in the right direction to approach design within current contexts of society.

Open-source projects would benefit from applying design justice principles especially when it comes to binary options and default assumptions that reinforce eurocentric ideals. Going forward, I would apply design justice to my own projects by restructuring my interpretation of goals with the community and not let my own biases dictate design choices. Also, I plan to be critical of the tools usd that may reinforce euro-centric ideals.

**Reply by zhamic7** [reply]
I think your point about intersectionality in the context of specific individuals is interesting! I guess I believe that intersectionality should ultimately be a principle used to acknowledge that the experiences of individuals cannot be categorized one-to-one into distinct identities and are rather instead shaped by a multitude of them. For example, I remember reading in the paper that courts ruled that companies weren't discriminating against either women or Black people, thus erasing the experiences of Black women. Personally liking or disliking a person based on the labels of identity they fit into, although unfortunately somewhat common today, seems like a different matter and a mischaracterization of what intersectionality is actually about.

**Reply by chrissanrow** [reply]
I haven't heard of such criticisms on intersectionality, but I definitely agree with the fact that it prioritizes the experience of the individual, which isn't an inherently bad thing. In the example you gave, intersectionality is being viewed largely on the basis of race and gender, but so many other identities (i.e. class, power) play into intersectionality in order to depict the experiences and position of a person or group of people. So, I think that in the example you gave, it is actually the abstraction away from intersectionality that leads to its flaws. In any case, I think it is still a necessary integration of design justice theory, as you also mentioned in your comment!

**Comment by nesslee123** [comment]
I believe design justice is incredibly important for addressing issues of inclusivity among design processes and creating encompassing and fulfilling projects. As discussed in the Costanza-Chock, S. (2018) article, ”Design is key to our collective liberation, but most design processes today reproduce inequalities that are structured by what Black feminist scholar Patricia Hill Collins calls the matrix of domination: white supremacy, heteropatriarchy, capitalism, and settler colonialism (p. 2). Due to this, it is imperative that we focus on reducing practices that cause oppression and inequality which is what design justice aims to do. This would be beneficial in open-source projects because it would allow for inclusivity amongst accessible projects which would allow for a larger and more diverse community to be involved with design justice and projects that involve design justice. Design Justice can and will be applied to my future projects, especially my final group project by ensuring that our survey and mapplication uplifts communities that are normally overlooked and highlights their experiences through our questions. Also, I would implement Design Justice into my projects by generally focusing on inclusivity, especially when it comes to marginalized groups.

**Reply by chrissanrow** [reply]
I like how you think of design justice as leading to fulfilling projects, as even though one of the pillars of design justice is to prioritize the design's impact on the community rather than our own intentions, it will certainly lead to a more fulfilling and impactful outcome.

**Comment by chrissanrow** [comment]
I think design justice is a critical theory that addresses large issues with hierarchies of power in not only technological tools, but also societal infrastructure and processes. It refocuses the design process to look at intersectional identities and consider, include, and empower those which are most prone to oppression given the particular design goals. Without having a critical mind when designing products, we may fall prey to "encoding" normative values and beliefs in what we create, jeopardizing historically and institutionally oppressed communities. As discussed in lecture, open-source projects are not exempt from ethical responsibility and can very much partake in this reinforcing of oppressive structures. Thus, an inclusive, mindful, empowering, and impact-driven design justice process is a key step in ensuring the ethicality of such projects. While as an individual, it may not be possible to implement all the pillars of design justice as Costanza-Chock outlines, I will try to be mindful of the guiding principles when ideating, designing, and developing projects. For instance, prioritizing impact on communities, centering voices of those who are directly impacted, and aiming to partake in an "accountable, accessible, and collaborative process" are all principles that could lead my future projects to success.

**Comment by kimada62** [comment]
Design justices questions equity in terms of a number of factors, but I thought the emphasis on the first three categories- who gets to design, who people design for, and what values are instilled in those designs, was the most impactful in the author’s tentative definition. Open source projects would most definitely benefit from utilizing these categories when considering what they are going to ask in their research questions and how they might choose to display that data. While I completed my project, I would say that our “who people design for” category was challenging. We were completing the project as part of a request from the San Jose Japanese American Museum who wanted a map of people’s most fond locations around the city. However, we had to keep in mind that we were mapping the underrepresented voices who don’t always have a platform first and foremost. It was not meant to be a project to represent the places, it was meant to be a project to show how we can better represent the community. Having the guidelines of this tentative design justice definition provides some good reminders of how one's platform could be built to fuel a social movement within their own neighborhood.
